src/Components/SearchForm.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export class SearchForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            searchQuery: props.searchQuery
+        };
+    }
+
+    handleInputChange = (event) => {
+        this.setState({ searchQuery: event.target.value });
+    };
+
+    handleButtonClicked = () => {
+        this.props.onSearch(this.state.searchQuery);
+    };
+
+    handleKeyDown = (event) => {
+        if (event.keyCode === 13) {
+            this.props.onSearch(this.state.searchQuery);
+        }
+    };
+
+    render() {
+        return (
+            <div className='searchFormContainer'>
+                <input
+                    placeholder='What do you want to watch?'
+                    type="text"
+                    value={this.state.searchQuery}
+                    onChange={this.handleInputChange}
```

Comment:
Some apps provide the functionality of searching while typing. If you need to do a similar implementation in the future, keep in mind a `debounce` or `throttle` functionality to avoid harming the backend with multiple requests.

Reply:
got it